chapters/part3/fairness.md

```diff
@@ -312,6 +312,8 @@ Fortunately, `allennlp.fairness` provides a `Model` wrapper called [`BiasMitigat
 
 2. `BiasMitigatorApplicator` works by applying a bias mitigator of your choice directly after the static embedding layer of the pretrained RoBERTA model during finetuning and evaluation. This reduces the model's ability to propagate gender biases across words in input sentences through contextualization.
 
+`BiasMitigatorApplicator` works with any Transformer model that has a static embedding layer. However, the relationships between (1) the number of epochs of finetuning, (2) the number of layers in the model, (3) the type of bias mitigator and bias direction used, and the efficacy of bias mitigation is not well-studied and a promising direction for future research. While finetuning for more epochs with the bias mitigator might push the model to depend less on gender, it could also cause the model to overfit to spurious correlations and/or discover proxies for gender that are not contained within the gender bias subspace of the mitigator. Furthermore, it is likely harder to mitigate biases for models with a larger number of layers, as they are capable of learning extremely complex mappings. And, how well a particular type of bias mitigator and bias direction work strongly depends on the finetuning data and model architecture. Fortunately, [empirical studies]((https://api.semanticscholar.org/CorpusID:201670701) have demonstrated that model performance on the evaluation set is not compromised by bias mitigation.
```

Comment:
This is a great draft for a paragraph.  I'd recommend one change: replace the last sentence with something like the following: "You should not assume that using a BiasMitigator resolves any particular bias issue in a model.  There is published literature suggesting that these techniques should help, but that needs to be evaluated for any actual downstream use."